app/Http/Controllers/PaginaBeheerController.php

```diff
@@ -95,6 +95,8 @@ private function validateData(Request $reqeust){
             'EN_text' => 'required',
             'content_nl' => 'required',
             'content_en' => 'required',
+            'parentItem' => 'required',
```

Comment:
Als een menu item in de menu balk moet komen te staan dan heeft die geen parentItem zo dit veld kan niet verplicht zijn.

Reply:
Aangepast dat die alleen verplicht is als `itemType` gelijk is aan `subItem`. Overigens stuurt de front-end dit veld altijd mee.